substrate/service/src/lib.rs

```diff
@@ -276,8 +273,6 @@ impl<Components> Drop for Service<Components> where Components: components::Comp
 	fn drop(&mut self) {
 		debug!(target: "service", "Substrate service shutdown");
 
-		self.network.stop_network();
```

Comment:
Doest the shutdown on Ctrl-C work OK after this? IIRC this was required so that network would terminate `Future` streams it exposes and that would release the BFT process and allow it to terminate.

Reply:
drop implements that same behaviour, so `drop(self.network.take());` in the following line should have the same effect.